src/runtime/GlobalObjectBuiltinString.cpp

```diff
@@ -907,21 +907,71 @@ static Value builtinStringToLocaleUpperCase(ExecutionState& state, Value thisVal
     return builtinStringToUpperCase(state, thisValue, argc, argv, isNewExpression);
 }
 
-static Value builtinStringTrim(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+static Value builtinStringTrimString(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
 {
+    // where parameters are the following
+    // 0 - start
```

Comment:
Please use an enum for it.

Reply:
That's a nice catch, gonna make the change in a minute.